config/locales/step_indicator/en.yml

```diff
@@ -4,7 +4,9 @@ en:
     accessible_label: Step progress
     flows:
       idv:
+        find_a_post_office: Find a Post Office
```

Comment:
These seem easy to miss later, should we add todo comments to revisit each google translation?

Reply:
@zachmargolis is there a standard practice for this? I've included Google Translate French and Spanish translations for the new steps.

Reply:
Not really 😬 maybe we create a JIRA ticket to track the keys? I forget if our YAML normalization comments stomp on comments, so I wouldn't necessarily rely on them